wagtail/wagtailadmin/menu.py

```diff
@@ -104,7 +105,7 @@ def __init__(self, label, menu, **kwargs):
 
     @property
     def media(self):
-        return Media(js=['wagtailadmin/js/submenu.js']) + self.menu.media
+        return Media(js=[static('wagtailadmin/js/submenu.js')]) + self.menu.media
```

Comment:
Djangos Media class already calls `static` on any paths defined, so wrapping the filename here in `static` is not needed: https://github.com/django/django/blob/master/django/forms/widgets.py#L80